src/customize-pop/accessibility-settings.md

```diff
@@ -0,0 +1,17 @@
+# Accessibility Settings
+
+Options to configure accessibility settings are located in `Settings` ➞ `Accessibility`. Accessibility features provide visual and audio enhancements to highlight system actions for greater usability. This section briefly describes each setting, but more information can be found on [GNOME's help page](https://help.gnome.org/users/gnome-help/stable/a11y.html).
+
+![Accessibility Settings](/images/accessibility-settings/accessibility-settings.png)
+
+## Always Show Accessibility Menu
+
+When the `Always Show Accessibility Menu` is enabled, accessibility options can be toggled using an icon in the upper right corner.
```

Comment:
This sounds like it's missing a word; it should either have another word after the blocked portion (like "When the `Always Show Accessibility Menu` option is enabled"), or it shouldn't have "the" before it (like "When `Always Show Accessibility Menu` is enabled").